packages/app/package.json

```diff
@@ -201,6 +201,7 @@
     "localforage": "^1.5.5",
     "lodash-es": "^4.17.2",
     "lru-cache": "^4.1.3",
+    "markty-toml": "^0.0.9",
```

Comment:
Last nitpick, could you move this to `common`?

Reply:
Done :D